Core/include/Acts/Seeding/Seedfinder.hpp

```diff
@@ -73,15 +73,15 @@ class Seedfinder {
   /// @param bottomSPs group of space points to be used as innermost SP in a seed.
   /// @param middleSPs group of space points to be used as middle SP in a seed.
   /// @param topSPs group of space points to be used as outermost SP in a seed.
-  /// @param rRangeSPExtent extent containing r values of all SP.
+  /// @param rMiddleSPRange vector containing min and max r variable range for middle SP.
   /// @note Ranges must return pointers.
   /// @note Ranges must be separate objects for each parallel call.
   template <template <typename...> typename container_t, typename sp_range_t>
   void createSeedsForGroup(
       State& state,
       std::back_insert_iterator<container_t<Seed<external_spacepoint_t>>> outIt,
       sp_range_t bottomSPs, sp_range_t middleSPs, sp_range_t topSPs,
-      Extent rRangeSPExtent) const;
+      Acts::Vector2 rMiddleSPRange) const;
```

Comment:
shouldn't that be `const Acts::Vector2 rMiddleSPRagne` ?

Reply:
yes, done!